examples/hooks-remix/.eslintrc

```
@@ -0,0 +1,3 @@
+{
+  "extends": ["@remix-run/eslint-config", "@remix-run/eslint-config/node"]
```

```suggestion
  "extends": ["@remix-run/eslint-config", "@remix-run/eslint-config/node"],
  "parserOptions": {
    "project": "../../tsconfig.json",
  }
```

I think this could fix this lint failure